google/cloud/sql/connector/connector.py

```diff
@@ -48,7 +49,12 @@ def _get_keys(loop: asyncio.AbstractEventLoop) -> concurrent.futures.Future:
     return _keys
 
 
-def connect(instance_connection_string: str, driver: str, **kwargs: Any) -> Any:
+def connect(
+    instance_connection_string: str,
+    driver: str,
+    ip_types: IPTypes = DEFAULT_IP_TYPE,
```

Comment:
I think DEFAULT_IP_TYPE is only used here, so we can just hardcode this to Public.